streampark-console/streampark-console-webapp/README.md

```diff
@@ -49,6 +49,19 @@ Open the project in Gitpod (free online dev environment for GitHub) and start co
 
 Console is written using Javascript. If you don't have a Node.js development environment, please [set it up](https://nodejs.org/en/download/). The minimum version required is 14.0.
 
+- Set the npm mirror source If you are experiencing a slow download of dependencies, you need to configure the npm mirror source by finding .npmrc in the user folder and writing the following
+
+```bash
+strict-peer-dependencies=false
+save-workspace-protocol = rolling
+registry = 'http://registry.npm.taobao.org'
+sass_binary_site='http://npm.taobao.org/mirrors/node-sass/'
```

Comment:
hi sizhu:
`http://registry.npm.taobao.org` Chinese developers access npm faster , so we can declare in README.zh-CN.md not README.md(developers from other countries don't know it. I recommend to set it as the official mirror of npm)